fourth review/swagger.yaml

```diff
@@ -0,0 +1,1366 @@
+        readOnly: true
+        enum:
+        - true
+        - false
+
+      total_tax:
+        type: string
+        description: Total tax amount.
+        readOnly: true
+
+      subtotal_price:
+        type: string
+        description: Subtotal after discounts, but before shipping and taxes have been applied.
+        readOnly: true
+
+      total_price:
+        type: string
+        description: Includes discounts, shipping and taxes.
+        readOnly: true
+
+      completed_at:
+        type: string
+        format: date-time
+        description: Date at which order is created and the draft order changes to “completed” status.
+        readOnly: true
+      created_at:
+        type: string
+        format: date-time
+        description: By default, this auto-generated property is the date and time when the draft order was created in Shopify, in ISO 8601 format. 
+        readOnly: true
+
+      updated_at:
+        type: string
+        format: date-time
+        description: The date and time when the draft order was last updated.
+        readOnly: true
+
+      status:
+        type: string
+        enum:
+        - open
+        - invoice_sent
+        - completed
+        description: String describing the state of the draft order.
+        readOnly: true
+
+  draft_order_invoice:
+    properties:
+      to:
+        type: string
+        description: >-
+          Email where the `draft_order_invoice` will be sent.  If left blank the email associated to the draft order is used instead; otherwise, the customer email is used.  If all three possible email sources are empty, an error is returned.
+          
+      from:
+        type: string
+        description: The email address that the email will appear to be sent from. This defaults to the store's customer email, and if none exists, the store's account email. When provided, this email address must correspond to the email address of one of the staff accounts.
+      bcc:
+        type: string
+        description: Optional array of email addresses to bcc. These must correspond to valid staff account email addresses.
+      subject:
+        type: string
+        description: The subject of the email that will be sent. When not provided, defaults to the Draft Order Invoice template's subject, as specified in the store's Notification settings.
+      custom_message:
+        type: string
+        description: An optional custom message provided in the body of the email.
+  checkout:
+    type: object
+    description: Object describing the checkout for the draft order.
+  line_item:
+    type: object
+    discriminator: custom
+   
+    properties:
+      custom:
+        description: Indicates if this is product_variant_line_item, or a custom_line_item.  If set to `true` indicates that the `line_item` is a custom line item.  If set to `false` indicates that it is a product variant `line_item`. *Read only field*
```

Comment:
* **product_variant_line_item** and **custom_line_item** should be in code tags?
*   "`true` indicates" -> "`true` it indicates" (same edit for "`false` indicates" later in this line)